universal-application-tool-0.0.1/app/views/components/Icons.java

```diff
@@ -39,6 +39,8 @@
           + " 100-2H7zm2 1a1 1 0 011-1h.01a1 1 0 110 2H10a1 1 0 01-1-1zm4-4a1 1 0 100 2h.01a1 1 0"
           + " 100-2H13zM9 9a1 1 0 011-1h.01a1 1 0 110 2H10a1 1 0 01-1-1zM7 8a1 1 0 000 2h.01a1 1 0"
           + " 000-2H7z";
+  public static final String PLUS_SVG_PATH =
+      "M10 5a1 1 0 011 1v3h3a1 1 0 110 2h-3v3a1 1 0 11-2 0v-3H6a1 1 0 110-2h3V6a1 1 0 011-1z";
```

Comment:
For my own learning -- how did you find/create this constant?

Reply:
I looked into the slack history and saw someone else ask :D The recommendation was to use https://heroicons.com/ or https://ikonate.com.  This is pulled from heroicons which the question type icons look to be from too.